src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -204,5 +204,12 @@
             </property>
             <deprecated>The service "%service_id%" is internal and deprecated since Symfony 3.3 and will be removed in Symfony 4.0</deprecated>
         </service>
+
+        <!-- Commands -->
+        <service id="form.command.debug" class="Symfony\Component\Form\Command\FormDebugCommand">
```

Comment:
I believe that's because all other debug commands are in the `FrameworkBundle`. Then it made sense for the framework bundle to wire its own commands itself.

But here, the form debug command is not tied at all to the framework. Maybe the wiring could be done in the DebugBundle then. 
But I'm not even convinced the debug commands should not be available when debug is false. It might still be appreciated to be able to run these in non-debug environments and does not have any downside to me.